x-pack/solutions/observability/plugins/exploratory_view/public/application/index.tsx

```diff
@@ -84,17 +83,15 @@ export const renderApp = ({
             }}
           >
             <Router history={history}>
-              <EuiThemeProvider>
-                <div className={APP_WRAPPER_CLASS} data-test-subj="exploratoryViewMainContainer">
-                  <RedirectAppLinks
-                    coreStart={{
-                      application: core.application,
-                    }}
-                  >
-                    <App startServices={core} />
-                  </RedirectAppLinks>
-                </div>
-              </EuiThemeProvider>
+              <div className={APP_WRAPPER_CLASS} data-test-subj="exploratoryViewMainContainer">
```

Comment:
@shahzad31 don't we need theme props in the App?

Reply:
So, no changes on the UI, Colors, etc..? Otherwise, you would need to share some screenshots so @patpscal can review it again